configure.ac

```diff
@@ -49,7 +49,7 @@ PKG_CHECK_MODULES(INITIAL_SETUP,
                   pango >= $PANGO_REQUIRED_VERSION
                   rest-0.7
                   json-glib-1.0
-		  libsecret-1
+                  libsecret-1
```

Comment:
@alepuccetti since these commits are minor fixes and not essential to carry downstream, I would just remove them from your branch. (That's what Cosimo means when he says "I'd prefer not to carry this downstream patch.") We'll have fewer downstream changes, and will get the fixes when we rebase on 3.26, whenever that happens. It's still good that you fixed these, of course. :)